lib/davinci_dtr_test_kit/profiles/communication_request_group.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'communication_request/communication_request_read'
+require_relative 'communication_request/communication_request_validation'
+
+module DaVinciDTRTestKit
+  class CommunicationRequestGroup < Inferno::TestGroup
+    title 'CRD CommunicationRequest Tests'
+    short_description 'Verify support for the server capabilities required by the CRD CommunicationRequest Profile'
+    description %(
+      # Background
+
+    The CRD CommunicationRequest sequence verifies that the system under test is
+    able to provide correct responses for CommunicationRequest queries. These queries
+    must return resources conforming to the [CRD CommunicationRequest Profile](http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-communicationrequest)
+    as specified in the DaVinci Coverage Requirements Discovery (CRD) v2.0.1
+    Implementation Guide.
+
+    # Testing Methodology
+    ## Read
+    First, Inferno will attempt to read each CommunicationRequest resource id provided in
+    the CommunicationRequest ids input. The server SHOULD be capable of returning a
```

Comment:
Nickpick "CommunicationRequest ids" is a bit different from the actual input name, which is "Communication Request IDs" (note the case of IDs).  Might as well be consistent here and in other tests.

Reply:
Addressed in [Correct IG links, wording, remove optional for some tests](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/20bdca6843a3ba2da4dfd0eb1800e5a326e5b768)